### PR TITLE
feat(zimbra): implement add autoreply page

### PR DESCRIPTION
**Pull request by tristanwagner**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-15840 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Related

Please merge https://github.com/ovh/manager/pull/13970 first, then this one thank you !

### CODE REVIEW

File: packages/manager/apps/zimbra/public/translations/autoReplies/add/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "zimbra_auto_replies_add_cta_back": "Retour vers mes répondeurs",
+  "zimbra_auto_replies_add_success_message": "Votre demande de création de répondeur a bien été prise en compte. Elle sera traitée d'ici quelques instants.",
+  "zimbra_auto_replies_add_error_message": "Votre demande de création de répondeur n'a pas pu aboutir. {{ error }}",
+  "zimbra_auto_replies_add_title": "Créer un répondeur",
+  "zimbra_auto_replies_add_header_create_for_account": "Vous êtes sur le point de créer un répondeur pour le compte",
+  "zimbra_auto_replies_add_header": "Veuillez saisir les informations du répondeur :",
+  "zimbra_auto_replies_mandatory_fields": "Les champs mentionnés avec un astérisque * sont obligatoires.",
+  "zimbra_auto_replies_add_account_label": "Adresse liée au répondeur",
+  "zimbra_auto_replies_add_input_email_placeholder": "",
```
Comment (ghyenne) [inline packages/manager/apps/zimbra/public/translations/autoReplies/add/Messages_fr_FR.json:10]:
```suggestion
```
Maybe it's no needed if stay empty.

File: packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AddAutoReply.page.tsx [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+  const [isFormValid, setIsFormValid] = useState(false);
+  const [selectedOrganizationId, setSelectedOrganizationId] = useState(
+    organizationId,
+  );
+
+  const goBackUrl = useGenerateUrl('..', 'path', params);
+
+  const goBack = () => {
+    return navigate(goBackUrl);
+  };
```
Comment (ghyenne) [inline packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AddAutoReply.page.tsx:105]:
```suggestion
  const goBack = () => navigate(goBackUrl);
```
We can use the short way.

File: packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AddAutoReply.page.tsx [hunk 3]
```diff
@@ -0,0 +1,640 @@
...
+              color={
+                form.until.hasError
+                  ? ODS_THEME_COLOR_INTENT.error
+                  : ODS_THEME_COLOR_INTENT.primary
+              }
+              value={untilDate}
+              minDate={fromDate || now}
+              onOdsDatepickerValueChange={(event) => {
+                handleFormChange('until', event.detail.formattedValue);
+              }}
```
Comment (ghyenne) [inline packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AddAutoReply.page.tsx:520]:
```suggestion
              onOdsDatepickerValueChange={(event) => handleFormChange('until', event.detail.formattedValue);}
```

File: packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AutoReplies.tsx [hunk 4]
```diff
@@ -63,21 +74,61 @@ const columns: DatagridColumn<AutoRepliesItem>[] = [
...
+  const { platformUrn } = usePlatform();
+  const [searchParams] = useSearchParams();
+  const params = Object.fromEntries(searchParams.entries());
+  const editEmailAccountId = searchParams.get('editEmailAccountId');
+  const hrefAddAutoReply = useGenerateUrl('./add', 'href', params);
+  const location = useLocation();
+
+  const shouldHide = useMemo(() => {
+    return location?.pathname?.endsWith('add');
+  }, [location]);
```
Comment (ghyenne) [inline packages/manager/apps/zimbra/src/pages/dashboard/AutoReplies/AutoReplies.tsx:89]:
```suggestion
  const shouldHide = useMemo(() => location?.pathname?.endsWith('add'), [location]);
```

**Review by antonymarion — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**